public/js/index.js

```diff
@@ -1 +1,93 @@
-console.log("hello world");
+const addButton = document.getElementById("js-button-add");
+const inputInForm = document.getElementById("js-input");
+const deleteButton = document.getElementById("js-delete-button");
+const fieldsWrapper = document.getElementById("js-fields-wrapper");
+let inputArray = document.querySelectorAll("#js-input");
+
+let indexPlus = 1;
+
+const getMainFormFromDom = () => {
+  return (mainForm = document.getElementById("js-form"));
+};
+
+const increaseIndex = () => {
+  indexPlus++;
+  //console.log(indexPlus);
+  return indexPlus;
+};
+const renderInput = () => {
+  return (innerHTML = `<div class="form__wrapper" id="js-form-input">
```

Comment:
jeśli wyrenderujesz kilka Inputów, to każdy będzie miał ten sam ID - a ID może być tylko 1 na stronie i musi być unikatowy

Reply:
dlaczego nie zwrócisz po prostu stringa, tylko przypisujesz go do jakieś globalnej zmiennej najpierw?